cmd/collector/app/builder/span_handler_builder.go

```diff
@@ -95,6 +96,7 @@ func (spanHb *SpanHandlerBuilder) initCassStore(builder cascfg.SessionBuilder) (
 		spanHb.collectorOpts.WriteCacheTTL,
 		spanHb.metricsFactory,
 		spanHb.logger,
+		casSpanstoreModel.DefaultTagFilter(),
```

Comment:
this would be better as a functional option, as the arg list is starting to get too long (and this one is clearly optional).